modules/db/src/main/scala/gem/dao/StaticConfigDao.scala

```diff
@@ -1,39 +1,90 @@
 package gem
 package dao
 
-// import doobie.imports._
+import doobie.imports._
 
 import gem.enum.Instrument
 import gem.config._
 
+import scalaz._
+import Scalaz._
+
 
 object StaticConfigDao {
 
-  // this is just a placeholder … right now the obs table just contains an instrument tag which we
-  // use to construct an empty static config
-
-  def forInstrument(i: Instrument): StaticConfig.Aux[i.type] =
-    (i match {
-      case Instrument.Phoenix    => PhoenixStaticConfig()
-      case Instrument.Michelle   => MichelleStaticConfig()
-      case Instrument.Gnirs      => GnirsStaticConfig()
-      case Instrument.Niri       => NiriStaticConfig()
-      case Instrument.Trecs      => TrecsStaticConfig()
-      case Instrument.Nici       => NiciStaticConfig()
-      case Instrument.Nifs       => NifsStaticConfig()
-      case Instrument.Gpi        => GpiStaticConfig()
-      case Instrument.Gsaoi      => GsaoiStaticConfig()
-      case Instrument.GmosS      => GmosSStaticConfig()
-      case Instrument.AcqCam     => AcqCamStaticConfig()
-      case Instrument.GmosN      => GmosNStaticConfig()
-      case Instrument.Bhros      => BhrosStaticConfig()
-      case Instrument.Visitor    => VisitorStaticConfig()
-      case Instrument.Flamingos2 => Flamingos2StaticConfig()
-    }).asInstanceOf[StaticConfig.Aux[i.type]] // Scala isn't smart enough to know this
+  def insert(s: StaticConfig): ConnectionIO[Int] =
+    for {
+      id <- Statements.insertBaseSlice(s.instrument).withUniqueGeneratedKeys[Int]("static_id")
+      _  <- insertConfigSlice(id, s)
+    } yield id
+
+  private def insertConfigSlice(id: Int, s: StaticConfig): ConnectionIO[Int] =
+    s match {
+      case _: AcqCamStaticConfig   => 0.point[ConnectionIO]
```

Comment:
It's a no-op.  I need a `ConnectionIO[Int]` to return but I don't really have anything to write to the database so I just put the value into a `ConnectionIO`.  I added cases for all the instruments so that I can get any observation to import as long as it has an instrument. The only one with an implementation at this point is F2.